### DISCUSSION TITLE
HSRB microphone output

### DISCUSSION THREAD

**Discussion by maltehue** [HumanSupportRobot/QandA#48] [Q&A] [answered]
Hello, i could use the interactive device check to verify that the hsrb can listen to what I say and repeat it, but i could not find any ROS or Pyhton interface to access the output of the microphone for further processing. I want to do the processing on an external development laptop.
Does anyone know how to do that?

**Comment by hawkina** [comment, marked as answer]
we ended up running audio_capture on the robot itself. 
See this for more details: http://wiki.ros.org/audio_common/Tutorials/Streaming%20audio
